example/pwa-assets.config.ts

```diff
@@ -1,22 +1,12 @@
 import {
```

Comment:
That's the thing, the user should not have a pwa-assets.config.ts because everything should be handled by the plugin and there should not be any more configuration.

Reply:
So let's remove that.

Reply:
what do you mean? the user can have that file to generate the assets using pwa assets CLI or use it on demand.

Maybe you can add a TODO.md file with the behavior of this plugin: I can revert changes to keep old options.